### DISCUSSION TITLE
pymupdf的版本问题

### DISCUSSION THREAD

**Discussion by cole-dda** [PaddlePaddle/PaddleOCR#12737] [Q&A]
请提供下述完整信息以便快速定位问题/Please provide the following information to quickly locate the problem

- 系统环境/System Environment：ubuntu18.04
- 版本号/Version：Paddle：  PaddleOCR： 问题相关组件/Related components：
   paddleocr==2.7.0
- 运行指令/Command Code：
- 完整报错/Complete Error Message：
 建议放宽pymupd的版本，因为在python3.11下没有wheel，每次安装需要编译，非常耗时
 PyMuPDF<1.21.0

**Comment by mikeshi80** [comment]
认同，1.22.5明明直接就提供了Win64的二进制包，如果这个`<1.21.0`的限制是没必要的话，则可以放开

**Comment by yaowt05** [comment]
是的，这就导致我在win64+python3.11下，pip install paddleocr会因为低版本的pymupdf编译失败而无法安装，而新版的pymupdf是提供了win64+python3.11的whl的，没必要依赖旧版pymupdf

**Comment by UserWangZz** [comment]
如果不影响使用，可以自己手动升级一下❤️
